config/routes/publish.rb

```diff
@@ -59,6 +59,7 @@
 
   get '/accept-terms', to: 'terms#edit', as: :accept_terms
   patch '/accept-terms', to: 'terms#update'
+  get '/suggestions', to: '/api/school_suggestions#index'
```

Comment:
this can go here https://github.com/DFE-Digital/publish-teacher-training/blob/1222-support-add-edit-accrediting-provider-step-1-non-js/config/routes/api.rb#L6

Reply:
Done